.circleci/infrastructure/commands.yml

```diff
@@ -67,3 +67,49 @@
       - terraform/apply:
           path: <<parameters.tf-path>>
           var_file: ./variables.tfvars
+
+  enable-s3-versioning:
+    parameters:
+      target_env:
+        type: string
+      cf-password:
+        type: env_var_name
+      cf-username:
+        type: env_var_name
+      cf-space:
+        type: string
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: |
+            sudo apt update
+            sudo apt install jq
+            sudo apt install curl
+            # TODO: Add Signature check
+            curl -L "https://packages.cloudfoundry.org/stable?release=linux64-binary&version=v7&source=github" | tar -zx
+            sudo mv cf7 /usr/local/bin/cf
+            sudo chmod +x /usr/local/bin/cf
```

Comment:
I see similar commands in `deploy-infrastructure`. Might be worth adding another command like `check-jq/curl`  alongside `sudo-check` and `cf-check` in the util folder and calling those. Minor optimization.

Reply:
Those are for more complicated install scripts. Sometimes we install curl and jq with apt sometimes with apk. We might be able to speed the pipeline up a little by making it all run on a custom alpine container that preserves resources better, but I think this is fine for now.